webrtc.html

```diff
@@ -5676,10 +5698,22 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                       track, without negotiating.</p>
                     </li>
                     <li>
-                      <p>Queue a task that sets <var>sender</var>'s
-                      <code><a data-for="RTCRtpSender">track</a></code>
-                      attribute to <var>withTrack</var>, and resolves
-                      <var>p</var> with <code>undefined</code>.</p>
+                      <p>Queue a task that runs the following steps:</p>
+                      <ol>
+                        <li>
+                          <p>If <var>connection</var>'s [[<a>isClosed</a>]]
+                          slot is <code>true</code>, abort these steps.</p>
+                        </li>
+                        <li>
+                          <p>Set <var>sender</var>'s <code><a data-for=
+                          "RTCRtpSender">track</a></code> attribute to
+                          <var>withTrack</var>.</p>
+                        </li>
+                        <li>
+                          <p>Resolve <var>p</var> with
+                          <code>undefined</code>.</p>
+                        </li>
+                      </ol>
```

Comment:
We do this in other places before promises are resolved, so it seems consistent.